fdbclient/NativeAPI.actor.cpp

```diff
@@ -2753,8 +2762,9 @@ Future<Void> Transaction::commitMutations() {
 			transactionSize = tr.transaction.mutations.expectedSize(); // Old API versions didn't account for conflict ranges when determining whether to throw transaction_too_large
 		}
 
-		if (transactionSize > (options.customTransactionSizeLimit == 0 ? (uint64_t)CLIENT_KNOBS->TRANSACTION_SIZE_LIMIT : (uint64_t)options.customTransactionSizeLimit))
+		if (transactionSize > options.customTransactionSizeLimit) {
```

Comment:
Should the parameter just be `sizeLimit` instead of `customTransactionSizeLimit`? Maybe there is a reason that's not obvious from the changed code, but I think this line looks weird now. (Like, if I just read this line, I might think, "why doesn't this handle the case where a custom size limit isn't set?".)

Reply:
Good suggestion. I was also thinking about changing it.